### PR TITLE
Update glossary entry for Quantity #30885

### PR DESCRIPTION
**Pull request by KhushPatibandha**
I closed last 2 PR for this issue cuz I had some conflict with my main branch, but anyways here is a new PR and I won't close this one, These are links for last 2 PRs if you want to compare them - https://github.com/kubernetes/website/pull/39867
https://github.com/kubernetes/website/pull/39856

### CODE REVIEW

File: content/en/docs/reference/glossary/quantity.md [hunk 1]
```diff
@@ -30,3 +30,18 @@ intentionally lowercase), `M` (mega), `G` (giga), `T` (tera), `P` (peta),
...
+                     `GB` stands for gigabyte, 
+                     `TB` stands for terabyte, 
+                     `PB` stands for petabyte, 
+                     `EB` stands for exabyte, 
+                     `KiB` stands for kibibyte, 
+                     `MiB` stands for mebibyte, 
+                     `GiB` stands for gibibyte, 
+                     `TiB` stands for tebibyte, 
+                     `PiB` stands for pebibyte, 
+                     `EiB` stands for exbibyte
```
Comment (sftim) [inline content/en/docs/reference/glossary/quantity.md:47]: I'd prefer to link to the a web page with the list. This list might make people think that these strings are valid as part of a quantity, but they are not.

Comment (KhushPatibandha) [reply content/en/docs/reference/glossary/quantity.md:47]: Hey, can you please elaborate more if possible?

**Comment by sftim** [comment]
Instead of putting a list of these suffixes here, hyperlink to a page that lists which suffixes are allowed in quantities.

**Don't** include the “B” (for byte), because that might mislead people.

**Comment by divya-mohan0209** [comment]
Hello @KhushPatibandha , gentle reminder. Would you please be able to address the suggestions by @sftim?

**Pull request closed without merging**